### PR TITLE
Bug 2070922 - [UI] new infra env view -> next steps: text enhancement

### PR DESCRIPTION
**Pull request by ammont82**
Related to https://bugzilla.redhat.com/show_bug.cgi?id=2070922
Signed-off-by: Montse Ortega <mortegag@redhat.com>

### CODE REVIEW

File: frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvForm.tsx [hunk 1]
```diff
@@ -90,12 +90,12 @@ const InfraEnvForm: React.FC<InfraEnvFormProps> = ({ control, handleChange }) =>
                             <CardBody>
                                 <Stack hasGutter>
                                     <StackItem>
-                                        Once you've successfully created your infrastructure environment go to the
-                                        details view and add hosts to it.
+                                        Once you successfully create your infrastructure environment, open the details
+                                        view and add hosts to it.
```
Comment (rawagner) [inline frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvForm.tsx:94]: according to BZ, this should be
```suggestion
                                        After your infrastructure environment is successfully created, open the
details view and click the "Add hosts" button.
```

**Review by rawagner — CHANGES_REQUESTED** [review]

File: frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvForm.tsx (hunk 1 above)
Comment (ammont82) [reply frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvForm.tsx]: It's true. Changed!

File: frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvForm.tsx [hunk 2]
```diff
@@ -90,12 +90,12 @@ const InfraEnvForm: React.FC<InfraEnvFormProps> = ({ control, handleChange }) =>
...
                                     <StackItem>
-                                        Once you've successfully created your infrastructure environment go to the
-                                        details view and add hosts to it.
+                                        After your infrastructure environment is successfully created, open the
+                                        details view and click the "Add hosts" button.
                                     </StackItem>
                                     <StackItem>
-                                        This will allow cluster creators to then pull from the infrastructure
-                                        environment any available hosts that have been added.
+                                        Adding hosts alows cluster creators to pull any available hosts from the
```
Comment (rawagner) [inline frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvForm.tsx:97]:
```suggestion
                                        Adding hosts allows cluster creators to pull any available hosts from the
```

**Review by rawagner — CHANGES_REQUESTED** [review]

File: frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvForm.tsx (hunk 2 above)
Comment (ammont82) [reply frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvForm.tsx]: I think now is ok! Thanks!

**Review by rawagner — APPROVED** [review]

**Comment by jtomasek** [comment]
`src/routes/Infrastructure/InfraEnvironments/InfraEnvForm.tsx` needs Prettier :)

**Review by jtomasek — APPROVED** [review]

**Pull request merged**